helpers/init_git_repo.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
I think merge commit (squash and merge GH button) is still fine for merging to master.
The purpose of rebasing on master is to get a better diff that shows the changes proposed by this PR rather than the two sided diff between master and the PR branch. The goal is to not (re)run tests that have already been merged upstream (e.g. from parallel PR recently merged that contained unrelated changes to this PR but will show up in the diff).
Inspiration from our friends over at https://github.com/GoogleCloudDataproc/initialization-actions/pull/640

Reply:
Got it.  Thanks for Clarifying.